elytron/src/main/resources/org/wildfly/extension/elytron/LocalDescriptions.properties

```diff
@@ -1139,7 +1139,10 @@ elytron.key-store.alias.certificate-chain.version=The certificate version.
 
 elytron.modifiable-key-store.read-alias=Read an alias from a KeyStore.
 elytron.modifiable-key-store.read-alias.alias=The alias of the KeyStore item to read.
+elytron.modifiable-key-store.read-alias.verbose=Instruct read to show HEX format fields content.
```

Comment:
Even with `verbose` set to `false`, there are still some HEX formatted fields in the output. So maybe we could reword this to something like:

Whether or not to include the public key and encoded form of a certificate in the output. The default value is true.